src/arguments.cpp

```diff
@@ -145,10 +145,10 @@ std::tuple<IpVer, std::string, uint8_t> Arguments::asIpAddrMask()
             if (ver.has_value())
             {
                 constexpr size_t ip4MaxMask = 32;
-                constexpr size_t ip6MaxMask = 128;
+                constexpr size_t ip6MaxMask = 64;
```

Comment:
In CIDR/VLSM this is called Prefix, not Mask. Mask is a kind of legacy term from classful networking times.

Reply:
Masks are always in form of x.x.x.x for IPv4.